app/models/noticed_notification.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class NoticedNotification < ApplicationRecord
```

Comment:
Keep this as notification

Reply:
`ActivityNotification` uses database `Notification`, we can change the table name after successful data migration and gem removal.
I tried that as well but was stuck in migration!